src/usecases/handle-figma-file-update-event-use-case.ts

```diff
@@ -8,8 +9,14 @@ import {
 } from '../infrastructure/repositories';
 
 export const handleFigmaFileUpdateEventUseCase = {
-	execute: async (webhookId: string, fileKey: string): Promise<void> => {
-		const figmaTeam = await figmaTeamRepository.getByWebhookId(webhookId);
+	execute: async (figmaTeam: FigmaTeam, fileKey: string): Promise<void> => {
+		const [connectInstallation, associatedFigmaDesigns] = await Promise.all([
+			connectInstallationRepository.get(figmaTeam.connectInstallationId),
+			associatedFigmaDesignRepository.findManyByFileKeyAndConnectInstallationId(
+				fileKey,
+				figmaTeam.connectInstallationId,
+			),
+		]);
 
 		// Ensure team admin OAuth2 credentials are still valid
```

Comment:
Just realised this block probably belongs to the `validateWebhookPayloadUseCase` now, will move it.